README.md

```diff
@@ -114,6 +114,22 @@ case 'rtf':
   return RtfViewer;
 ```
 
+### Viewing local changes in a secondary repo
+
+If you are working on a feature branch and need to see changes introduced in that branch in another repo that using this library, here are the steps:
+
+1. Run `yarn build` in this react-file-viewer repo
+2. In the secondary repo, update `package.json` to point the trussworks/react-file-viewer declaration to your branch in the react-file-viewer repo:
+
+```"@trussworks/react-file-viewer": "git+https://github.com/trussworks react-file-viewer#your-branch-name"
```

Comment:
You'll need to add a newline for this to be displayed properly. (Ie. have the three distinct lines, two for the beginning and ending ``` and one for the code itself.)